subworkflows/nf-core/fastq_align_dedup_bwamem/main.nf

```diff
@@ -23,18 +27,46 @@ workflow FASTQ_ALIGN_DEDUP_BWAMEM {
     ch_multiqc_files                 = Channel.empty()
     ch_versions                      = Channel.empty()
 
-    FASTQ_ALIGN_BWA (
-        ch_reads,
-        ch_bwamem_index,
-        true, // val_sort_bam hardcoded to true
-        ch_fasta
-    )
-    ch_alignment        = ch_alignment.mix(FASTQ_ALIGN_BWA.out.bam)
-    ch_alignment_index  = ch_alignment.mix(FASTQ_ALIGN_BWA.out.bai)
-    ch_stats            = ch_alignment.mix(FASTQ_ALIGN_BWA.out.stats)    // channel: [ val(meta), path(stats) ]
-    ch_flagstat         = ch_alignment.mix(FASTQ_ALIGN_BWA.out.flagstat) // channel: [ val(meta), path(flagstat) ]
-    ch_idxstats         = ch_alignment.mix(FASTQ_ALIGN_BWA.out.idxstats) // channel: [ val(meta), path(idxstats) ]
-    ch_versions         = ch_versions.mix(FASTQ_ALIGN_BWA.out.versions.first())
+    if (use_gpu) {
+        /*
+        * Align with parabricks GPU enabled fq2bammeth implementation of bwameth
+        */
+        PARABRICKS_FQ2BAM (
+            ch_reads,
+            ch_fasta,
+            ch_bwamem_index,
+            interval_file, // interval file
+            known_sites, // known sites
+            'bam' // output format

```

Comment:
I just checked with the other modules, and I think they are all compatible with `'cram'`. So this shouldn't be hardcoded to `'bam'` but instead should also be passed down as an input param.